subsys/bluetooth/host/hci_core.h

```diff
@@ -6,6 +6,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define HCI_CMD_TIMEOUT		K_SECONDS(10)
+
```

Comment:
Isn't the move of this define unnecessary now?

Reply:
Yes, I will move it back again.